parser/Dockerfile

```
@@ -1,8 +1,12 @@
-FROM python:3
+FROM python:3.12-slim
```

.python-versionと合わせた方が良さそう
```suggestion
FROM python:3.13-slim
```